### PR TITLE
bsdlib: version 0.7.6

### PR DESCRIPTION
**Pull request by kpuusaari**
* Added bsdlib support for TLS_CIPHERSUITE_LIST. List of supported
  ciphersuites can be obtained using getsockopt(), and selecting
  one is done by setsockopt().
* Support for sending more than 2k packets in TLS socket.

Signed-off-by: Kimmo Puusaari <kimmo.puusaari@nordicsemi.no>

### CODE REVIEW

**Comment by kpuusaari** [comment]
Manifest: 
https://github.com/nrfconnect/sdk-nrf/pull/2604

File: bsdlib/doc/CHANGELOG.rst [hunk 1]
```diff
@@ -5,6 +5,14 @@ Changelog
 
 All notable changes to this project are documented in this file.
 
+bsdlib 0.7.6
+************
+
+* Added bsdlib support for ``TLS_CIPHERSUITE_LIST``. List of supported
+  ciphersuites can be obtained using :cpp:func:`getsockopt()`, and
+  selecting one is done by :cpp:func:`setsockopt()`.
```
Comment (umapraseeda) [inline bsdlib/doc/CHANGELOG.rst:13]:
```suggestion
* Added bsdlib support for ``TLS_CIPHERSUITE_LIST``.
  :cpp:func:`getsockopt()` lists the supported cipher suites and :cpp:func:`setsockopt()` selects a supported cipher suite.
```

File: bsdlib/doc/CHANGELOG.rst [hunk 2]
```diff
@@ -5,6 +5,14 @@ Changelog
...
+* Support for sending more than 2k packets in TLS socket.
```
Comment (umapraseeda) [inline bsdlib/doc/CHANGELOG.rst:14]:
```suggestion
* Support for sending packets sized more than 2048 bytes in TLS socket.
```

File: bsdlib/doc/CHANGELOG.rst (hunk 1 above)
Comment (kpuusaari) [reply bsdlib/doc/CHANGELOG.rst]: changed

File: bsdlib/doc/CHANGELOG.rst (hunk 2 above)
Comment (kpuusaari) [reply bsdlib/doc/CHANGELOG.rst]: changed

**Pull request merged**